src/libs/OptionsListUtils.js

```diff
@@ -77,16 +78,17 @@ Onyx.connect({
 });
 
 const lastReportActions = {};
-const allReportActions = {};
+const allSortedReportActions = {};
 Onyx.connect({
     key: ONYXKEYS.COLLECTION.REPORT_ACTIONS,
     callback: (actions, key) => {
         if (!key || !actions) {
             return;
         }
-        allReportActions[key] = actions;
+        const sortedReportActions = ReportActionUtils.getSortedReportActions(_.toArray(actions), true);
         const reportID = CollectionUtils.extractCollectionItemID(key);
-        lastReportActions[reportID] = _.last(_.toArray(actions));
+        allSortedReportActions[reportID] = sortedReportActions;
+        lastReportActions[reportID] = _.first(sortedReportActions);
     },
```

Comment:
I'm surprised this doesn't impact any other methods here, there isn't any place that uses allReportActions???

Reply:
Also how come the change to lastReportActions? It doesn't seem like it is used in any of your changes

Reply:
Yeah a couple things were surprising when I started looking at this, as you mentioned.

1. allReportActions is unused (idk why lint didn't catch that)
2. lastReportActions are not actually the "last" report actions for each report. They don't come pre-sorted in the object, they need to be sorted as we do on the ReportScreen with getSortedReportActions. I figured I should fix that while I'm making changes.